src/speech_to_text_finetune/data_process.py

```diff
@@ -35,6 +35,7 @@ def process_dataset(
     dataset: DatasetDict,
     feature_extractor: WhisperFeatureExtractor,
     tokenizer: WhisperTokenizer,
+    num_proc: int | None,
```

Comment:
I would probably just set this to `os.cpu_count()` by default

Reply:
good idea! done